From 3666c07ffd4126bf8fa8b2f6ac12fe4e96548348 Mon Sep 17 00:00:00 2001
From: tangjie133 <1334875503@qq.com>
Date: Wed, 22 Feb 2023 11:28:17 +0800
Subject: [PATCH] change code

---
 src/DFRobot_EnvironmentalSensor.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/DFRobot_EnvironmentalSensor.h b/src/DFRobot_EnvironmentalSensor.h
index 71dcf12..4cd30f0 100644
--- a/src/DFRobot_EnvironmentalSensor.h
+++ b/src/DFRobot_EnvironmentalSensor.h
@@ -15,7 +15,7 @@
 #include "Arduino.h"
 #include "Wire.h"
 #include "DFRobot_RTU.h"
-#include "String.h"
+#include "string.h"
 
 #if (defined ARDUINO_AVR_UNO) && (defined ESP8266)
 #include "SoftwareSerial.h"
